UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/PlayFabMultiplayerAgentView.cs

```diff
@@ -7,8 +7,34 @@ namespace PlayFab
 
     public class PlayFabMultiplayerAgentView : MonoBehaviour
     {
+        /// <summary>
+        /// Static reference to the current active instance of the PlayFabMultiplayerAgentView.
+        /// </summary>
+        public static PlayFabMultiplayerAgentView current { get; private set; };
```

Comment:
public static PlayFabMultiplayerAgentView current { get; private set; } = null;
I like to be explicit in assignments so it is clear something wasn't forgotten.